components/header/header.hbs

```diff
@@ -1,8 +1,63 @@
-<div class="container">
-  <header class="header">
-      <div class="logo header__logo">
-          <img src="https://cdn.ons.gov.uk/assets/images/ons-logo/v2/ons-logo.svg" alt="Office for National Statistics" class="logo__img">
-      </div>
-  </header>
-</div>
-{{>@bar bar_title="eQ Title bar"}}
+<header class="header {{classes}}">
+    <div class="header__top" role="banner">
+        <div class="container">
+            <div class="grid grid--gutterless">
+                <div class="grid__col col-6@s">
+                    <div class="logo">
+                        <a href="#1"><img src="{{path '/assets/img/ons-logo-white.svg'}}" alt="Office for National Statistics logo" class="logo__img header__logo"></a>
+                    </div>
+                </div>
+                {{#if services}}
+                <div class="grid__col col-6@s">
+                    <nav class="nav nav--horizontal nav--light u-d-no--@xs@s u-fr" aria-label="Site services menu">
+                        <ul class="nav__list" role="menubar" aria-label="Site services menu">
+                            {{#each services}}
+                            <li class="nav__item" role="menuitem"><a href="{{link}}" class="nav__link">{{text}}</a></li>
+                            {{/each}}
+                        </ul>
+                    </nav>
+                </div>  
+                {{/if}}
+            </div>
+        </div>
+    </div>
+    <div class="header__main">
+        <div class="container">
+            <div class="grid grid--gutterless grid--align-mid u-cf">
+                <div class="grid__col {{#if nav}}col-10@xs col-6@s col-8@m{{/if}}">
+                    <h1 class="header__title">{{title}}</h1>
+                    {{#if description}}
+                    <p class="header__desc">{{description}}</p>
+                    {{/if}}
+                </div>
+                {{#if nav}}  
+                {{> @button--mobile btn-classes='u-fr u-d-no--@s'}}
+                {{/if}}
+                {{#if button}}
+                <div class="u-fr u-d-no--@xs@s">
+                    {{> @button btn-classes='btn--ghost print__hidden' label=button}}
+                </div>
+                {{/if}}
+            </div>
+        </div>
+    </div>
+    {{#if nav}}  
+    <div class="header__nav">
+        <div class="container">
+            <nav class="nav nav--horizontal nav--light nav--main nav--h-m js-main-nav" aria-label="Main menu" id="main-nav">
```

Comment:
In **Voiceover** this menu is announced irrespective of whether it is in use on these pages.

https://deploy-preview-111--sdc-global-design-patterns.netlify.com/components/preview/header-navigation--secondary.html

and

https://deploy-preview-111--sdc-global-design-patterns.netlify.com/components/preview/header-navigation--save.html

The main menu is read out in Voiceover after the `H1`.

`<nav ... aria-label="Main menu"` should not be included if no content is available.